draft-irtf-cfrg-opaque.md

```diff
@@ -1491,7 +1489,7 @@ Output:
 - session_key, the shared session secret.
 
 Exceptions:
-- HandshakeError, when the handshake fails
+- InvalidServerAkeMACError, the handshake fails.
```

Comment:
Can we rename this to "HandshakeAuthenticationError", "ServerAuthenticationFailure," or similar? I don't know if we need to note "AKE" and "MAC".

Reply:
Isn't that more telling?
I fear that ServerAuthenticationFailure is too vague and doesn't say what the actual error is. e.g. it could be mistaken with the server sending the wrong public key (or whatever - this would result in the same error). I like the precision of this wording to pinpoint what went wrong.

Reply:
Maybe ServerAuthenticationMACError then?